models.py

```diff
@@ -9,6 +9,13 @@
 from app_core import app, db
 from utils import generate_key
 
+from flask_admin.babel import lazy_gettext
```

Comment:
move to other flask_admin imports

Reply:
can you give me an example of how?

Reply:
moved this with the other flask_admin something.